bulbs/utils/percolator.py

```diff
@@ -0,0 +1,70 @@
+import re
+
+from elasticsearch_dsl.connections import connections
+
+from bulbs.content.models import Content
+from bulbs.sections.models import Section
+from bulbs.special_coverage.models import SpecialCoverage
+
+
+PERCOLATOR_PATTERNS = {
+    'section.(\w+)': Section,
+    'specialcoverage.(\w+)': SpecialCoverage
+}
+
+
+def get_query_id_list(es, index, doc_type):
+    """Searches an elasticsearch index for all results of a given doc_type"""
+
+    results = es.search(index=index, doc_type=doc_type)
+    results_hits = results.get('hits', None)
+
+    if results_hits is None:
```

Comment:
probably better to do `if not result_hits:`, in case ES results ever return empty dict or list